core/api/gen.go

```diff
@@ -12,16 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package core
+package api
 
-//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --go_out=,plugins=grpc:$GOPATH/src api/v1/keytransparency_proto/keytransparency.proto api/v1/keytransparency_proto/admin.proto api/v1/keytransparency_proto/mutations.proto
-//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --grpc-gateway_out=logtostderr=true:. api/v1/keytransparency_proto/keytransparency.proto api/v1/keytransparency_proto/admin.proto api/v1/keytransparency_proto/mutations.proto
+//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --go_out=,plugins=grpc:$GOPATH/src v1/keytransparency_proto/keytransparency.proto v1/keytransparency_proto/admin.proto v1/keytransparency_proto/mutations.proto
+//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --grpc-gateway_out=logtostderr=true:. v1/keytransparency_proto/keytransparency.proto v1/keytransparency_proto/admin.proto v1/keytransparency_proto/mutations.proto
 
-//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --go_out=,plugins=grpc:$GOPATH/src api/monitor/v1/monitor_proto/monitor.proto
-//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --grpc-gateway_out=logtostderr=true:. api/monitor/v1/monitor_proto/monitor.proto
+//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --go_out=,plugins=grpc:$GOPATH/src monitor/v1/monitor_proto/monitor.proto
+//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --grpc-gateway_out=logtostderr=true:. monitor/v1/monitor_proto/monitor.proto
 
-//go:generate protoc -I=. -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis --go_out=:$GOPATH/src api/usermanager/v1/usermanager_proto/usermanager.proto
```

Comment:
what happened to this line?

Reply:
I did some experiments and discovered that the following two lines are redundant, since the grpc invocation will also compile the proto message objects. 
- `--go_out=:$GOPATH/src`
- `--go_out=,plugins=grpc:$GOPATH/src`

Reply:
Cool, thanks for explaining.